src/nvim/os/unix_defs.h

```diff
@@ -2,7 +2,7 @@
 #define NVIM_OS_UNIX_DEFS_H
 
 // Windows doesn't have unistd.h, so we include it here to avoid numerous
-// instances of `#ifdef HAVE_UNISTD_H'.
+// instances of `#ifdef WIN32'.
 #include <unistd.h>
```

Comment:
The only reason to include these though is to account for platform differences.

I think it's only more confusing in that `HAVE_IO_H` would only be used to pull in `<io.h>` on Windows and vice-versa.